tools/run_nested_shell.py

```
@@ -168,6 +168,28 @@ def run_shell(
         subprocess.run((gnome_shell, *shell_args), env=shell_env, check=True)
 
 
+def run_shell_devkit(*, env, **kwargs):
```

Ruff (0.14.10)

Missing type annotation for `**kwargs`

(ANN003)